youtube_dl/extractor/shared.py

```diff
@@ -86,6 +93,12 @@ class VivoIE(SharedBaseIE):
         },
     }
 
+    def _extract_title(self, webpage):
+        stream_content = extract_attributes(self._search_regex(
```

Comment:
There is totally no point in `extract_attributes`. Just fix the regex.

Reply:
I did not see what was wrong with the regex, so I looked at how other extractors implement similar problems. To me, `extract_attributes` seemed to be a common pattern.

Reply:
@dstftw May I ask you to review my new change? I think I fixed the regex.